src/main/java/jenkins/scm/impl/subversion/SubversionSCMSource.java

```diff
@@ -159,6 +165,14 @@ public String getCredentialsId() {
     public void setCredentialsId(String credentialsId) {
         this.credentialsId = credentialsId;
     }
+    @DataBoundSetter
+    public void setAdditionalCredentials(List<AdditionalCredentials> additionalCredentials) {
+        this.additionalCredentials = additionalCredentials;
+    }
+    //without getter jelly will crash
+    public List<AdditionalCredentials> getAdditionalCredentials() {
+        return(additionalCredentials);
```

Comment:
FindBugs will grumble about exposing external implementation. Either make it `@Restricted` for Jelly only or wrap with `unmodifyableList`